test/common/ops/changeClass.js

```diff
@@ -70,13 +70,9 @@ describe('shared.ops.changeClass', () => {
 
       expect(user.stats.class).to.equal('rogue');
       expect(user.flags.classSelected).to.be.true;
-      expect(user.items.gear.equipped.weapon).to.equal('weapon_rogue_0');
       expect(user.items.gear.owned.weapon_rogue_0).to.be.true;
-      expect(user.items.gear.equipped.armor).to.equal('armor_rogue_1');
       expect(user.items.gear.owned.armor_rogue_1).to.be.true;
```

Comment:
I think this `expect` for armor_rogue_1` could be removed, along with `user.items.gear.owned.armor_rogue_1 = true;` above. I believe their only purpose was to test that the armor was equipped when it was already owned, which is no longer relevant.